org.videolan.VLC.Plugin.makemkv.json

```diff
@@ -75,10 +54,10 @@
         "--disable-gui",
         "--prefix=/app/share/vlc/extra/makemkv",
         "DESTDIR=/",
-        "PKG_CONFIG_PATH=/app/share/vlc/extra/makemkv/ffmpeg/lib/pkgconfig"
+        "PKG_CONFIG_PATH=/app/share/vlc/extra/makemkv/ffmpeg/lib/pkgconfig",
```

Comment:
As far as I understand, since the ffmpeg block from above got removed, the file `/app/share/vlc/extra/makemkv/ffmpeg/lib/pkgconfig` no longer exists? Is there any point in setting `PKG_CONFIG_PATH` to a non-existent file, i.e. does the build fail if you omit this line?

Reply:
indeed, I'll remove it thanks.